pkg/codegen/templates/typedef.tmpl

```diff
@@ -1,5 +1,5 @@
 {{range .Types}}
-// {{.TypeName}} defines model for {{.JsonName}}.
+// {{.TypeName}} {{ with .Description }}represents {{ . }}{{ else }}defines model for {{.JsonName}}{{ end }}.
```

Comment:
If I define a description, I would find it strange to have "represents" added in a hard coded manner. What do you think about just taking the description {{.}} as is?

Reply:
You're right, I've changed it. Thanks!